### PR TITLE
Add Discord recipe

### PR DESCRIPTION
**Pull request by lucasmezencio**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A

Add Discord recipe. 🙂

### CODE REVIEW

**Comment by joubertredrat** [comment]
This sound good, I use deployer and have discord server, then, why not?

File: src/Discord/Messaging.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Deployer\Discord;
```
Comment (antonmedv) [inline src/Discord/Messaging.php]: Can it be done without using extra classes?

Comment (lucasmezencio) [reply src/Discord/Messaging.php]: Hey @antonmedv! I think yes, but why?
I mean, I think that separating things could be a good way to organise the code.
What do you think?
🙂

Comment (antonmedv) [reply src/Discord/Messaging.php]: Yes, but not v6 have some problems with autoloading and phars. I'm going to solve this in future version bout to work correctly on all version of deployer now must not include composer autoload.

Comment (lucasmezencio) [reply src/Discord/Messaging.php]: @antonmedv got you! Thanks!

I'll update my code then. :)

**Comment by lucasmezencio** [comment]
Hey @antonmedv do you think it's doable to do something like this? 🙂

**Comment by lucasmezencio** [comment]
@antonmedv hey, news? 🙂

**Comment by antonmedv** [comment]
Please, remove MessagingInterface and all classes, it's redundant. :)

**Comment by lucasmezencio** [comment]
@antonmedv done!

**Pull request merged**